packages/block-library/src/query-pagination-next/index.php

```diff
@@ -51,7 +51,7 @@ function render_block_core_query_pagination_next( $attributes, $content, $block
 				'<a href="%1$s" %2$s>%3$s</a>',
 				esc_url( add_query_arg( $page_key, $page + 1 ) ),
 				$wrapper_attributes,
-				esc_html( $label )
+				$label
```

Comment:
Ideally I'd prefer to escape at the point where we're building the markup. But because of the potential concat of the pagination arrow it's difficult. If anyone wants to refactor to still allow us to escape as close to the point of output as possible that would be preferable.

Reply:
To escape as late as possible wouldn't it be preferable to use `wp_kses`?

```
wp_kses( $label,
	array(
		'span' => array(
			'class' => array(),
		),
	)
)
```